amplium/specs/swagger.yml

```diff
@@ -68,6 +68,22 @@ paths:
           schema:
             $ref: '#/definitions/ok_response'
 
+  /proxy/api/session/{session_id}:
```

Comment:
I'm confused now what url should I type in the browser's address bar to get the status. is it `mhcadminproxy-ci.aws.wgen.net/amplium/proxy/api/session/{session_id}`?

Reply:
or it's not intended for humans even?

Reply:
We still have `/status` endpoint, which gives us info about running Selenium Grid.
The new endpoint can get us info about a specific Selenium node, which is associated with Selenium session_id.
Let's say, I create a new session by starting tests, and I would like to know where my tests are running by that session id, so, if something breaks, I can debug/ ssh to the node.